src/Scaffold/Console/model/model.stub

```diff
@@ -22,6 +22,28 @@ class {{studly_name}} extends Model
      */
     protected $fillable = [];
 
+    /**
+     * @var array Rules for fields
+     */
+    protected $rules = [];
```

Comment:
Rules is a public property not protected and it requires the validation trait to be included.